### DISCUSSION TITLE
什么是发件箱模式 | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#30] [Announcements]
# 什么是发件箱模式 | 连波的闲谈杂鱼

早上在群里看到有同事在问关于 Outbox Pattern 的问题，而在此之前我对此一无所知，我甚至还错误的发了一些毫不相干的回复，那就姑且把 Outbox Pattern 当作今天 TIL 的对象吧，下面是本次学习的记录

https://godruoyi.com/posts/what-is-the-outbox-pattern/

**Comment by myxiaoao** [comment]
如果是 MySQL，阿里的 canal 也可以。

**Reply by godruoyi** [reply]
谢谢，又了解了一个东西。好奇那时候是 debezium 还没发布还是别的什么原因呢？为什么 ali 要自己搞一个并且还只能适配 MySQL 的方案。

**Reply by myxiaoao** [reply]
因为阿里去 IOE，用的 MySQL 来代替的 Oracle 。

**Reply by godruoyi** [reply]
哇，谢谢，去 IOE 原来是 `去掉IBM的小型机、Oracle数据库、EMC存储设备`
